src/ducks/filters/index.js

```diff
@@ -64,7 +64,7 @@ export const getTransactionsFilteredByAccount = createSelector(
     if (accountIds) {
       transactions = filterByAccountIds(transactions, accountIds)
     }
-    return transactions
+    return sortBy(transactions, t => -new Date(t.date))
```

Comment:
If find it's not very obvious that it sorts from most recent to most ancient. Maybe putting the result in a variable with a clear name before returning it would help reading.